docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
+2. User accesses logs
+3. User looks for a stacktrace with their App’s processes name
+4. User finds stacktrace + root cause and begins to fix
+5. User commits fix
+6. User’s Build passes
+7. User merges PR
+#### Use Case
+1. GitHub has Status update and comment from fastlane on PR indicating that the build failed
+2. User opens fastlane ci Web UI
+4. Assuming already logged in
+5. User opens Project for their Repo + branch
```

Comment:
Reminder:
`Project` is:
- single repo
- 1 or 2 triggers, "Manual" by default, and 1 and only 1 of these [`nightly`, `commit`]
- Triggers contain a reference to a `branch` which is not used at this moment.